apps/ledger-live-desktop/tests/handlers/nfts.ts

```diff
@@ -0,0 +1,9 @@
+import { http, HttpResponse } from "msw";
+
+const handlers = [
+  http.post("https:/example", () => {
+    return HttpResponse.json([]);
+  }),
```

Comment:
why ?

Reply:
It was just to keep a trace of it but yes I will delete it